src/main/java/com/minecolonies/coremod/colony/buildings/workerbuildings/BuildingLumberjack.java

```diff
@@ -378,6 +420,78 @@ public void removeCitizen(final ICitizenData citizen)
         super.removeCitizen(citizen);
     }
 
+    /**
+     * Bonemeal fungi in netherTrees
+     */
+
+    private void bonemealFungi()
+    {
+        for (final BlockPos pos : getNetherTrees())
+        {
+            final World world = colony.getWorld();
+            if (WorldUtil.isBlockLoaded(world, pos))
+            {
+                BlockState blockState = world.getBlockState(pos);
+                Block block = blockState.getBlock();
+                if (block == Blocks.CRIMSON_FUNGUS || block == Blocks.WARPED_FUNGUS)
+                {
+                    int threshold = getMainCitizen().getCitizenSkillHandler().getLevel(getPrimarySkill()) * FUNGI_MODIFIER;
```

Comment:
That one is a bit annoying probably, but I feel we should have a curve here. Like, there should always be a 10% chance to grow the tree. With increasing level it should then get to 100%, instead of very low chance early on and 100% after level 50

Reply:
this will also error when there is no citizen for this building, check if we have a citizen first